lib/core/start.js

```diff
@@ -56,11 +56,21 @@ function start (events) {
 		var ssl = keystone.get('ssl');
 		var unixSocket = keystone.get('unix socket');
 		var startupMessages = ['KeystoneJS Started:'];
+		var lex = keystone.get('letsencrypt');
 
```

Comment:
Could we rename this to something that signifies what it is? Maybe `letsencrypt`? :grin: